libs/net/dal/Models/ContentFilter.cs

```diff
@@ -45,6 +45,11 @@ public class ContentFilter : PageFilter
     /// </summary>
     public int? SourceId { get; set; }
```

Comment:
Can we get rid of the duplication?  Just use the array.

Reply:
I might create another (small) ticket for this as it results in some rework to out Advanced Search